public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,30 @@
+// Scripts for firebase and firebase messaging
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js');
+
+// Initialize the Firebase app in the service worker by passing the generated config
+var firebaseConfig = {
+    apiKey: 'AIzaSyDEoVx6711d0M50SVUmXhejuO8shhzFC8I',
+    authDomain: 'photorater-c5c6a.firebaseapp.com',
+    projectId: 'photorater-c5c6a',

```

Comment:
Codacy has a fix for the issue: [Replace `··projectId:·'photorater-c5c6a'` with `projectId:·"photorater-c5c6a"`](https://app.codacy.com/gh/albertnahas/Lingwars/pullRequest?prid=9567702)
```suggestion
  projectId: "photorater-c5c6a",
```